builtin/plugins/user_deployer_whitelist/user_deployer_whitelist.go

```diff
@@ -260,11 +260,21 @@ func (uw *UserDeployerWhitelist) RemoveUserDeployer(ctx contract.Context, req *u
 	if err := contract.CallMethod(ctx, dwAddr, "RemoveUserDeployer", removeUserDeployerRequest, nil); err != nil {
 		return errors.Wrap(err, "failed to remove deployer")
 	}
-	// remove from current contract deployer address
-	if !ctx.Has(deployerStateKey(deployerAddr)) {
-		return ErrDeployerDoesNotExist
+	if ctx.FeatureEnabled(loomchain.UserDeployerWhitelistVersion1_1Feature, false) {
```

Comment:
This feature has already gone out in the latest production build, so we'll need to use a new one here named `UserDeployerWhitelistVersion1_2Feature`

Reply:
done